src/app/modules/bookmark/component/bookmark-settings/bookmark-settings.component.ts

```diff
@@ -1,4 +1,4 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core'
+import { ChangeDetectionStrategy, Component, ElementRef, Input, TemplateRef, ViewChild } from '@angular/core'
```

Comment:
There seem to be some redundant imports here.

Reply:
yea i'll remove the unused TemplateRef